spec/project_spec.rb

```diff
@@ -723,18 +723,36 @@ class InvalidErrorHandler; end
             allow(project_instance.decision_service).to receive(:get_variation_for_feature).and_return(decision_to_return)
 
             expect(project_instance.get_feature_variable_string('boolean_single_variable_feature', 'boolean_variable', user_id, user_attributes))
-              .to eq('true')
+              .to eq(nil)
 
-            expect(spy_logger).to have_received(:log).twice
+            expect(spy_logger).to have_received(:log).once
```

Comment:
Why two lines with?
  expect(spy_logger).to have_received(:log).once

Reply:
Done!